.rubocop_todo.yml

```diff
@@ -14,6 +14,7 @@ Lint/ConstantDefinitionInBlock:
     - 'lib/puppet/provider/network_config/interfaces.rb'
     - 'lib/puppet/provider/network_config/redhat.rb'
     - 'lib/puppet/provider/network_route/routes.rb'
+    - 'lib/puppet/provider/network_route/redhat.rb'
```

Comment:
please do not add more exceptions to the file. This is already some technical debt that we need to cleanup and it would be nice if this wouldn't grow.

Reply:
well im not sure what action to take here the reason i had to add exceptions is because i have copied behavior from network_config over to network_route.

It's the only way i can remove the nil exception in filesync that occurs due to some strange behavior in the other module that generates a null named resource

Reply:
ok this is fixed